work/covidvu/pipeline/vuhospitals.py

```diff
@@ -0,0 +1,56 @@
+import urllib
+import json
+import pandas as pd
+from tqdm.auto import tqdm
+from covidvu.pipeline.vujson import _dumpJSON
+
+import os
+from os.path import join
+
+from os.path import join
+from time import sleep
+
+# *** constants ***
+STATE_CODES_PATH = join(os.getcwd(), 'stateCodesUS.csv')
+SITE_DATA               = './site-data'
```

Comment:
This is defined in one of the modules, I believe covidvu.pipeline.vujson - let's fix it later.  I'll merge/approve this.

Reply:
Yep, you're right.